apps/web/src/pages/Departments/components/DepartmentTable.tsx

```diff
@@ -29,7 +30,7 @@ export const DepartmentTable = ({
   const columns = [
     columnHelper.accessor("departmentNumber", {
       id: "departmentNumber",
-      filterFn: "equals",
+      filterFn: "weakEquals",
```

Comment:
Department number searching wasn't working and this seemed to fix it. Seems it was a case of comparing a number to a stringified number, so needs to not match type